pom.xml

```diff
@@ -194,57 +128,8 @@
 						<appLocation>${project.artifactId}.war</appLocation>
 					</bootstrapProperties>
 				</configuration>
-				<executions>
-					<execution>
-						<id>install-liberty</id>
-						<phase>prepare-package</phase>
-						<goals>
-							<goal>install-server</goal>
-						</goals>
-					</execution>
-					<execution>
-						<id>package-app</id>
-						<phase>package</phase>
-						<goals>
-							<goal>package-server</goal>
-						</goals>
-					</execution>
-				</executions>
 			</plugin>
 			<!-- Plugin to run functional tests -->
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-failsafe-plugin</artifactId>
-				<version>2.18.1</version>
-				<executions>
-					<execution>
-						<phase>integration-test</phase>
-						<id>integration-test</id>
-						<goals>
-							<goal>integration-test</goal>
-						</goals>
-						<configuration>
-							<includes>
-								<include>**/it/**</include>
```

Comment:
Surefire is for unit test, so no it is not needed for the test provided, but Surefire looks for anything that starts or ends with `Test`.  The war plugin automatically binds surefire, so you don't want anything in */it/ to start or end in `Test` or surefire WILL try to run it even after you remove it from your pom.xml. 
Although TestApplication.java looks empty, surefire will still try to run it.